drkafka/src/main/java/com/pinterest/doctorkafka/config/DoctorKafkaConfig.java

```diff
@@ -43,6 +47,9 @@
   private static final String NOTIFICATION_EMAILS = "emails.notification";
   private static final String ALERT_EMAILS = "emails.alert";
   private static final String WEB_BIND_HOST = "web.bindhost";
+  public static final String DRKAFKA_ADMIN_ROLE = "drkafka_admin";
```

Comment:
@yuyang08 this constant isn't a config but the name of the role used inside Security context, ideally it should be separated by - or _ characters to avoid confusion